src/components/Navbar.js

```diff
@@ -12,8 +12,19 @@ const NavBar = () => {
       <li>
         <a href="jsx-a11y/anchor-is-valid">Homepage</a>
       </li>
-    </ul>
+    </NavBarStyles>
   );
 };
 
-export default NavBar
\ No newline at end of file
+const NavBarStyles = styled.ul`
+  justify-content: center;
+  align-items: center;
+  display: Flex;
+  height: 70px;
+  position: fixed;
+  bottom: 0;
+  margin: 0 0 0 0;
```

Comment:
Same here for margin as above mentioned.

Reply:
Done.